proto/v1/config.proto

```diff
@@ -229,6 +230,14 @@ message Hook {
     string webhook_url = 1 [json_name="webhookUrl"];
     string template = 2 [json_name="template"];
   }
+
+  message Beeep {
+    string template = 1 [json_name="template"];
+    string title_template = 2 [json_name="titleTemplate"];
+    double frequency = 3 [json_name="frequency"];
```

Comment:
Im somewhat ambivalent about accepting a frequency and duration , I wonder if most users mind or would rather have some opinionated default e.g. simply an enum. 

I could see something like

```
enum DeliveryMode {
   SILENT,
   LOW_PITCH_CHIME,
   HIGH_PITCH_CHIME,
}
```
. 

WDYT? are you finding the range of frequency control useful?